docs/faq/General.md

```diff
@@ -71,3 +72,22 @@ Redux is originally written in ES6 and transpiled for production into ES5 with W
 > The new Counter Vanilla example is aimed to dispel the myth that Redux requires Webpack, React, hot reloading, sagas, action creators, constants, Babel, npm, CSS modules, decorators, fluent Latin, an Egghead subscription, a PhD, or an Exceeds Expectations O.W.L. level.
 
 >Nope, it's just HTML, some artisanal `<script>` tags, and plain old DOM manipulation. Enjoy!
+
+
+<a id="general-when-to-learn"></a>
+### When should I learn Redux?
+
+What to learn can be an overwhelming question for a JavaScript developer. It helps to narrow the range of options by focusing on problems you find in your work. Redux is a pattern for managing application state. If you do not have problems with state management, you might not need to learn Redux. Especially if you are new to JavaScript application development, we encourage you to start with a library (like React) for creating data-driven user interfaces. Such a library might be all you need to build your application. If your application becomes so complex that you are confused about where state is stored or how state changes, then it might be a good time to learn Redux. 
```

Comment:
Pretty good start.  Got a couple ideas for rephrasing things.

First, let me quote my "standard advice for learning React":

> On that note, definitely don't over-complicate the learning process by trying to learn many different things at once. Some people will say you should use a "boilerplate" to learn React, and they're wrong - boilerplate projects almost always come with too many pieces configured, and are confusing for beginners.

> Instead, the best advice is to focus on learning React itself first. Once you have a good understanding of how React works, you will better appreciate why a state management library like Redux can be useful, and you can learn about other tools later.

I'd like to see if we can use some of that phrasing.  It's not so much that React is a "data-driven UI library", but that React already has a built-in pattern for managing state.  

I definitely like that last sentence on "becoming so complex you're confused".

Another aspect I'd like to get into this is that we advise learning React first just so there's fewer concepts to understand at once.  It's certainly possible for people to tackle them both at the same time, but for most people it's easier to focus on React first, then Redux after that.

Reply:
I've revised this to be more explicit that a UI library is possibly sufficient when the library includes a state management system. And I've added emphasis that beginners are best off starting without Redux. I'm reluctant to comment on React more specifically because it undermines the important point that Redux is not coupled to React. Saying "focus on React first, then Redux after that" could imply a preference for React more fundamentally. So I tried to incorporate the principles of what you're saying with less emphasis on React.